containers/daap-update-metadata/src/var/task/update_metadata.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+from http import HTTPStatus
+
+import boto3
+from data_platform_api_responses import format_error_response, format_response_json
+from data_platform_logging import DataPlatformLogger
+from data_product_metadata import DataProductMetadata, InvalidUpdate
+
+s3_client = boto3.client("s3")
+
+
+logger = DataPlatformLogger(
+    extra={
+        "image_version": os.getenv("VERSION", "unknown"),
+        "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+    }
+)
+
+
+def handler(event, context):
+    data_product_name = event["pathParameters"].get("data-product-name")
+    table_name = event["pathParameters"].get("table-name")
```

Comment:
is this needed to update the version within the metadata object?

Reply:
Oh no, this shouldn't be here at all. I'll remove it